server/src/main/java/org/eclipse/lsp/cobol/core/engine/dialects/idms/IdmsDialect.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2022 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package org.eclipse.lsp.cobol.core.engine.dialects.idms;
+
+import org.antlr.v4.runtime.CharStreams;
+import org.antlr.v4.runtime.CommonTokenStream;
+import org.eclipse.lsp.cobol.core.IdmsLexer;
+import org.eclipse.lsp.cobol.core.IdmsParser;
+import org.eclipse.lsp.cobol.core.engine.dialects.CobolDialect;
+import org.eclipse.lsp.cobol.core.engine.dialects.DialectOutcome;
+import org.eclipse.lsp.cobol.core.engine.dialects.DialectParserListener;
+import org.eclipse.lsp.cobol.core.messages.MessageService;
+import org.eclipse.lsp.cobol.core.model.ResultWithErrors;
+import org.eclipse.lsp.cobol.core.model.SyntaxError;
+import org.eclipse.lsp.cobol.core.model.tree.Node;
+import org.eclipse.lsp.cobol.core.strategy.CobolErrorStrategy;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/** Processing the text according to the IDMS rules */
+public final class IdmsDialect implements CobolDialect {
+  public static final String NAME = "IDMS";
+
+  /**
+   * Processing the text according to the IDMS rules
+   *
+   * @param uri document URI
+   * @param text document text
+   * @param messageService error message service
+   * @return the dialect processing result
+   */
+  public ResultWithErrors<DialectOutcome> processText(String uri, String text, MessageService messageService) {
+    IdmsLexer lexer = new IdmsLexer(CharStreams.fromString(text));
```

Comment:
You should remove the default listeners of the Lexer to avoid the pollution of the STDOUT

Reply:
Sure.